docs/features/security/secrets/auto-invalidate-secrets.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Automatically Invalidate Azure Key Vault Secrets"
+layout: default
+---
+
+# Automatically Invalidate Azure Key Vault Secrets
+
+The `Arcus.WebApi.Jobs` library provides a background job to automatically invalidate cached Azure Key Vault secrets from an `ICachedSecretProvider` instance of your choice.
+
+## How This Works
+
+This automation works by subscribing on the `SecretNewVersionCreated` event of an Azure Key Vault resource and placing those events on a Azure Service Bus Topic; which we process in our background job.
+
+To make this automation opperational, following Azure Resources has to be used:
+* Azure Key Vault instance
+* Azure Service Bus Topic
+* Azure Event Grid subscription for `SecretNewVersionCreated` events that are sent to the Azure Service Bus Topic
+
+## Usage
+
+Our background job has to be configured in `ConfigureServices` method:
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+    // An 'ISecretProvider' implementation to access the Azure Service Bus Topic resource;
+    //     this will get the 'serviceBusTopicConnectionStringSecretKey' string (configured below) and has to retrieve the connection string for the topic.
+    var mySecretProvider = new MySecretProvider();
```

Comment:
Good remark by Gillian, folks don't have to implement themselves so I would just refer to arcus security.